inventory_management_system_api/routers/v1/usage_status.py

```diff
@@ -0,0 +1,75 @@
+"""
+Module for providing an API router which defines routes for managing Usage statuses using the `UsageStatusService`
+service.
+"""
+
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, status, HTTPException, Path
+
+from inventory_management_system_api.core.exceptions import DuplicateRecordError, InvalidObjectIdError
+from inventory_management_system_api.schemas.usage_status import UsageStatusPostRequestSchema, UsageStatusSchema
+from inventory_management_system_api.services.usage_status import UsageStatusService
+
+logger = logging.getLogger()
+
+router = APIRouter(prefix="/v1/usage-statuses", tags=["usage statuses"])
+
+
+@router.post(
+    path="",
+    summary="Create a new usage status",
+    response_description="The created usage status",
+    status_code=status.HTTP_201_CREATED,
+)
+def create_usage_status(
+    usage_status: UsageStatusPostRequestSchema,
+    usage_status_service: Annotated[UsageStatusService, Depends(UsageStatusService)],
+) -> UsageStatusSchema:
+    # pylint: disable=missing-function-docstring
+    logger.info("Creating a new usage status")
+    logger.debug("Usage status data: %s", usage_status)
+
+    try:
+        usage_status = usage_status_service.create(usage_status)
+        return UsageStatusSchema(**usage_status.model_dump())
+
+    except DuplicateRecordError as exc:
+        message = "A usage status with the same name already exists"
```

Comment:
```suggestion
        message = "A usage status with the same value already exists"
```
Just thought this is actually the value.